src/languages/c-like.js

```diff
@@ -105,6 +105,7 @@ export default function(hljs) {
       'atomic_bool atomic_char atomic_schar ' +
       'atomic_uchar atomic_short atomic_ushort atomic_int atomic_uint atomic_long atomic_ulong atomic_llong ' +
       'atomic_ullong new throw return ' +
+      'class struct' +
```

Comment:
These keywords were missing generally: there are many situations (not just templates) in which they are valid and weren’t currently highlighted; `class/struct foo` is valid wherever `foo` is used as a type name, such as in declarations or in casts (`foo y = (class foo)(x)`).

Reply:
What often happens (and why these were excluded here) is false positives on `bob.class()`... does C++ allow class or struct as method names on classes?

Reply:
It does not: C++ keywords are reserved in *all* contexts. There are some context-dependent reserved words (e.g. `override` and `final`) but these aren’t part of this list.